### DISCUSSION TITLE
"Pull Labs": Ideas for future apps that can make use of our principles and codebase

### DISCUSSION THREAD

**Discussion by ethanblake4** [PullDating/Org#114] [Ideas & Features]
Given we opt to design Pull as a 'platform' as in https://github.com/PullDating/Org/discussions/38 (already being done in the Flutter app), here's some of my ideas for future sub-brand apps targeted towards specific niches. Feel free to comment with more. This is all "future future".

- Two: Video-focused dating app where after matching, there is no option to text; instead, you are able only to video chat when you are both online.
- Lipsync: Link your spotify, get matched with people based on your music taste
- Hyper: The main UI of the app is basically a message board, where people can post specific things they want to do and when, eg "I want to go see Dr. Strange on Friday evening". Other users can scroll through and match with ones they find interesting.
- Crush: For people you know IRL. You choose one or more of your contacts that you like, they get an anonymous text message that one of their contacts has a crush on them, then they do the same and if you both choose each other the match is revealed. Very easy to go viral, no obvious path to revenue.
- Flutter: Not really a dating app, just an app for finding travel buddies based on where you want to go and when. (No relation to the Flutter SDK.)

**Comment by Oshyan** [comment]
Wow, I kind of love all these ideas! Awesome.

**Comment by WilliamDormer** [comment]
This was an idea of one of the people that joined: An app for double dates, so that people (women) feel safer meeting up in person.

**Comment by ajbott** [comment]
These are all brilliant! I'd love to see all these features be implemented into app "events" / experiments.
